std/assembly/math.ts

```diff
@@ -3132,3 +3137,27 @@ export function ipow64f(x: f64, e: i32): f64 {
   }
   return sign ? 1.0 / out : out;
 }
+
+function seedRandomSelect(): i64 {
+  if (isDefined(ASC_SEEDRANDOM_FUNC)) return ASC_SEEDRANDOM_FUNC();
+  if (isDefined(ASC_SEEDRANDOM_MATH)) {
+    let val: i64;
+    do val = reinterpret<i64>(JSMath.random());
+    while (!val);
+    return val;
+  }
+  if (isDefined(ASC_SEEDRANDOM_DATE)) return <i64>JSDate.now();
+  if (isDefined(ASC_SEEDRANDOM_WASI)) {
+    let buf = __alloc(8, 0);
```

Comment:
First 8 bytes usually zeros and never contain meaningful information. What about use this first bytes instead allocation and zero back before return `buf`?

Reply:
I guess we could do that, but some simple modules might have an empty memory with zero pages. Not sure

Reply:
just check `memory.size`?

Reply:
or use conditional compilation